ionoscloud/resource_k8s_cluster.go

```diff
@@ -103,6 +103,31 @@ func resourcek8sCluster() *schema.Resource {
 					"to `false` and should not be provided otherwise.",
 				Optional: true,
 			},
+			"api_subnet_allow_list": {
+				Type: schema.TypeList,
+				Description: "Access to the K8s API server is restricted to these CIDRs. Cluster-internal traffic is not " +
+					"affected by this restriction. If no allow list is specified, access is not restricted. If an IP " +
+					"without subnet mask is provided, the default value will be used: 32 for IPv4 and 128 for IPv6.",
+				Optional: true,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
+			},
+			"s3_buckets": {
+				Type:        schema.TypeList,
+				Description: "List of S3 bucket configured for K8s usage. For now it contains only an S3 bucket used to store K8s API audit logs.",
+				Optional:    true,
+				Computed:    true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"name": {
+							Type:        schema.TypeString,
```

Comment:
what is the reason for not having a list of strings here, are we planning for more properties on the `s3_buckets` items?